subworkflows/nf-core/fastq_align_bwaaln/meta.yml

```diff
@@ -46,6 +47,10 @@ output:
       description: |
         Files containing software versions
         Structure: [ path(versions.yml) ]
+  - //:
```

Comment:
This also seems broken... @mirpedrol ?

Reply:
This is but in tools, probably because of a multiline comment, thanks for pointing this out!